recipes/delphes/meta.yaml

```
@@ -0,0 +1,70 @@
+{% set name = "delphes" %}
+{% set version = "3.5.0" %}
```

try at least https://github.com/delphes/delphes/commit/64de5a479b94beba357e73c926cdc6ee9114ceec for this to see if this works better.

```suggestion
{% set version = "3.5.1pre10" %}
```

---

I'd prefer to start the feedstock at the last known version and then patch our way to passing. We can make a secondary release with `v3.5.1pre10` later but I think I'd also just like to ask the Delphes maintainers for a new release.

---

ok.